src/Microsoft.ML.Transforms/Text/TextTransform.cs

```diff
@@ -262,6 +262,30 @@ public static IDataTransform Create(IHostEnvironment env, Arguments args, IDataV
                 view = new ConcatTransform(h, new ConcatTransform.Arguments() { Column = xfCols }, view);
             }
 
+            if (tparams.NeedsNormalizeTransform)
```

Comment:
@justinormont, I moved `TextNormalizer` above `WordTokenizer`. Let me know if there is any adverse effect of it?

Reply:
I don't like this change. It's far from clear to me that if someone asks to remove stopwords that they meant for the content of stopwords to be retained in the chargrams. At the very least this ought to be a configurable option.

---
In reply to: [204000786](https://github.com/dotnet/machinelearning/pull/548#discussion_r204000786) [](ancestors = 204000786)

Reply:
@TomFinley, your comment is not clear to me. Can you elaborate more on this? Don't you see it good to have `TextNormalizer` applied before `WordTokenizer`?

---
In reply to: [204201054](https://github.com/dotnet/machinelearning/pull/548#discussion_r204201054) [](ancestors = 204201054,204000786)

Reply:
Sorry thought you were saying something else for some reason got confused.